systemtest/070-list-tags.bats

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bats
+#
+# list-tags tests
+#
+
+load helpers
+
+function setup() {
+    standard_setup
+
+    start_registry --enable-delete=true reg
+}
+
+# list from registry
+@test "list-tags: remove image from registry" {
+    local remote_image=docker://quay.io/libpod/busybox:latest
+    local img_tag=unsigned
+    local localimg=docker://localhost:5000/busybox:${img_tag}
+    local output=
+
+    run_skopeo copy --dest-tls-verify=false $remote_image $localimg
+    output=$(run_skopeo inspect --tls-verify=false --raw $localimg)
+    echo $output | grep "vnd.docker.distribution.manifest.v2+json"
```

Comment:
Please don't do this. If it fails, the error message will be completely worthless:
```
    `echo $output | grep "vnd.docker.distribution-manifest.v2+json"' failed         
```
(it gives no indication of what the problem was). Instead, use the existing `expect_output` mechanism:
```
    run_skopeo inspect --tls-verify=false --raw $localimg
    mediatype=$(jq -r '.mediaType' <<<"$output")
    expect_output --from="$mediatype" \
                  "application/vnd.docker.distribution.manifest.v2+json"
```
(the `jq` tightens up the test, making it less likely to false-positive on something else in the `inspect` output)